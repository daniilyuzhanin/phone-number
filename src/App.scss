.wrapper {
  display: flex;
  align-items: center;
  justify-content: center;
  min-height: 100vh;
  flex-direction: column;

  background: #e8cbc0;
  background: -webkit-linear-gradient(to right, #636fa4, #e8cbc0);
  background: linear-gradient(to right, #636fa4, #e8cbc0);
}
