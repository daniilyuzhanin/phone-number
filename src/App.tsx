import './App.scss';
import { PhoneForm } from 'common/components/PhoneForm';
import { PhoneTable } from 'common/components/PhoneTable';

export const App = () => {

  return (
    <div className="wrapper">
      <PhoneForm />
      <PhoneTable />
    </div>
  );
}