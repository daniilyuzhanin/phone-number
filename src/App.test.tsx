import { render } from '@testing-library/react';

import { App } from './App';

test('renders home page', () => {
  const { container } = render(
    <App />
  );
  expect(container).toMatchSnapshot();
});