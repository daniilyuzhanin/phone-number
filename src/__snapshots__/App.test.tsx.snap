// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page 1`] = `
<div>
  <div>
    Hello!
  </div>
</div>
`;
